.github/workflows/generate.yml

```diff
@@ -0,0 +1,30 @@
+name: Generate
+
+on:
+  workflow_dispatch: # Allows manual triggering of the workflow to generate SDK
+    inputs:
+      force:
+        description: "Force generation of SDKs"
+        type: boolean
+        default: false
+  schedule:
+    - cron: 0 0 * * * # Runs every day at midnight
+  push:
+    branches:
+      - '*'
+jobs:
+  generate:
+    uses: speakeasy-api/sdk-generation-action/.github/workflows/sdk-generation.yaml@v14
+    with:
+      speakeasy_version: latest
+      openapi_doc_auth_header: Authorization
+      openapi_docs: |
+        - https://raw.githubusercontent.com/Alramech/speakeasy-test/main/openapi/openapi.yaml
```

Comment:
When do we swap this with the `edge` open api yaml?

Reply:
Ideally we'd do it when we move the openapi out of edge into it's own single file repo

Reply:
We should swap this to `edge` before we ship. Possible to do it with PAT? Is it an issue with private repositories? If so, wouldn't we have the same issue whether this spec lives inside `edge` or another repo?

Reply:
I've figured out how to generate a PAT for edge but have hit a minor generation issue around connections, waitinf on resolution with the team.